### DISCUSSION TITLE
Dynamic trusted IP list and kong hot reload

### DISCUSSION THREAD

**Discussion by shettyh** [Kong/kong#8654] [Help] [answered]
Hi,

Is there any way to do kong hot reload when there is update to the kong config ( trusted ip list ) ?

One option is to add a file watcher script that basically triggers `kong reload`. Is there any cleaner way to do this ?

Thanks

**Comment by hbagdi** [comment, marked as answer]
That is your only option as far as I know.

**Reply by shettyh** [reply]
Ok, Thanks for the response

**Reply by shettyh** [reply]
@hbagdi is there any reference that i can check for using `inotify` for config file watch with kong inside a container ?

**Reply by hbagdi** [reply]
I've done that using systemd before and it worked fine. I don't have a reference handy.

**Reply by shettyh** [reply]
Ok thanks
